### ISSUE TITLE
Countries data not returning

### ISSUE THREAD

**Issue by GeorgeFive** [tboothman/imdbphp#133]
Title basically sums it up. Script is not returning country data since last night. It doesn't appear that anything has changed on their end at a glance, but who knows.

**Comment by duck7000** [comment]
rating and photo doesn't work either

**Comment by tboothman** [comment]
Yea there's a lot of failures: https://travis-ci.org/tboothman/imdbphp/jobs/416459896

**Comment by tboothman** [comment]
Interestingly there's now a load of JSON in the page .... that should make this a lot easier:
```
{
  "@context": "http://schema.org",
  "@type": "TVSeries",
  "url": "/title/tt3032476/",
  "name": "Better Call Saul",
  "image": "https://m.media-amazon.com/images/M/MV5BMTAxOTQ0MjUzMzJeQTJeQWpwZ15BbWU4MDY0NTAxNzMx._V1_.jpg",
  "genre": [
    "Crime",
    "Drama"
  ],
  "contentRating": "TV-MA",
  "actor": [
    {
      "@type": "Person",
      "url": "/name/nm0644022/",
      "name": "Bob Odenkirk"
    },
    {
      "@type": "Person",
      "url": "/name/nm0781533/",
      "name": "Rhea Seehorn"
    },
    {
      "@type": "Person",
      "url": "/name/nm0052186/",
      "name": "Jonathan Banks"
    },
    {
      "@type": "Person",
      "url": "/name/nm0264579/",
      "name": "Patrick Fabian"
    }
  ],
  "creator": [
    {
      "@type": "Person",
      "url": "/name/nm0319213/",
      "name": "Vince Gilligan"
    },
    {
      "@type": "Person",
      "url": "/name/nm0332467/",
      "name": "Peter Gould"
    },
    {
      "@type": "Organization",
      "url": "/company/co0212797/"
    },
    {
      "@type": "Organization",
      "url": "/company/co0520966/"
    },
    {
      "@type": "Organization",
      "url": "/company/co0071947/"
    },
    {
      "@type": "Organization",
      "url": "/company/co0086397/"
    },
    {
      "@type": "Organization",
      "url": "/company/co0019701/"
    }
  ],
  "description": "The trials and tribulations of criminal lawyer, Jimmy McGill, in the time leading up to establishing his strip-mall law office in Albuquerque, New Mexico.",
  "datePublished": "2015-02-08",
  "keywords": "lawyer,brother brother relationship,male protagonist,shared universe,albuquerque new mexico",
  "aggregateRating": {
    "@type": "AggregateRating",
    "ratingCount": 222864,
    "bestRating": "10.0",
    "worstRating": "1.0",
    "ratingValue": "8.7"
  },
  "review": {
    "@type": "Review",
    "itemReviewed": {
      "@type": "CreativeWork",
      "url": "/title/tt3032476/"
    },
    "author": {
      "@type": "Person",
      "name": "Kieran Battams"
    },
    "dateCreated": "2015-02-13",
    "inLanguage": "English",
    "name": "Works not only as a spin off.. but as a show in its own right!",
    "reviewBody": "Breaking Bad in my opinion is the perfect show and is my all-time favourite and Walter White\u0027s seedy lawyer Saul Goodman has always been one of my favourite characters of the show. The second i found out he would be getting his own spin off i was excited and Better Call Saul has been one of the most hotly anticipated releases of 2015. I have to say based on the 2 episodes that have aired so far I have to say it has not disappointed and if the show continues to be this good then I will have another perfect show in my watchlist.\n\nIt has been great to see Saul Goodman (Jimmy McGill as he is known here) back to his roots, here we see a very different version of the character as a down on his luck lawyer as opposed to the criminal lawyer who handed Jesse Pinkman bags of dollars without a care in the world, yet his charm and his cocky persona and sometimes hilarious one liners stay exactly where we want them. The tone also feels very different to breaking bad. We get to see a much more lighthearted approach to this series.\n\nWe also get glimpses of Mike Ermantraut (Jonathan Banks) which was great and whilst so far we haven\u0027t seen a lot of him i am certain we will get a lot more interaction between the two (he is second billed in the titles afterall). I was also surprised to see another fan favourite although i wont spoil this for those who don\u0027t know.\n\nAs the series goes on i will post about my opinions more but i feel this show is great and will continue to go up and be on par with Breaking Bad. Vince Gilligan has proved himself as a force to be reckoned with. Better Call Saul is not only awesome as a spin off, but in its own right as a show also. Better Call Saul is undeniably awesome",
    "reviewRating": {
      "@type": "Rating",
      "worstRating": "1",
      "bestRating": "10",
      "ratingValue": "9"
    }
  },
  "trailer": {
    "@type": "VideoObject",
    "name": "Season 4: Official Comic-Con Trailer",
    "embedUrl": "/video/imdb/vi1556462361",
    "thumbnail": {
      "@type": "ImageObject",
      "contentUrl": "https://m.media-amazon.com/images/M/MV5BZmEzZmI2MzgtYzRmZC00ZjU4LWI0NTItNjBlOGVhNWQwMGYwXkEyXkFqcGdeQXRzdGFzaWVr._V1_.jpg"
    },
    "thumbnailUrl": "https://m.media-amazon.com/images/M/MV5BZmEzZmI2MzgtYzRmZC00ZjU4LWI0NTItNjBlOGVhNWQwMGYwXkEyXkFqcGdeQXRzdGFzaWVr._V1_.jpg",
    "description": "There’s rough road ahead for Jimmy and Mike. Don’t miss the Season Premiere, Monday, August 6."
  }
```

**Comment by jreklund** [comment]
It dosen't contain everything. So we need to pick and choose from it.

**Comment by jreklund** [comment]
@tboothman Are you going to do these or should I? So that I don't start on it then I come back.

**Comment by tboothman** [comment]
I'll be doing the stuff that's in json-ld

**Comment by jreklund** [comment]
All done! We have now released v6.0.3.
